### ISSUE TITLE
When using VoiceOver on text, it is read with English accent. On a number, it is read out in English.

### ISSUE THREAD

**Issue by tochwill** [BBC-archive/psammead#3307]
**Describe the bug**
When using voiceover, numbers are read out in english rather than in the service language. And text is read with an english accent.

- This should be in the service language.
- However - is this just because when testing this, the user who is performing the test does not have that language installed on their machine?

If that is the case, close this issue as this is expected behaviour.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/hausa?renderer_env=live
2. Using VoiceOver, navigate to the radio schedules section.
3. See error

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
We may need to test this with a device OS in the same language as the page, does anyone in the team speak/understand any of the languages we have this on test for?

**Comment by OlgaLyubin** [comment]
I have added fake Russian data for Radio Schedule locally and changed my Mac's language to Russian. All of the numbers (start time and duration) were read out in Russian language in VoiceOver. Also, text was not read with an English accent.

The only thing that was read out in English was the word "Duration", but that is a know issue since we don't have translations yet.

<img width="1126" alt="Снимок экрана 2020-04-01 в 17 37 49" src="https://user-images.githubusercontent.com/17802955/78234980-db07df80-74cf-11ea-8505-111a103e6167.png">

**Comment by OlgaLyubin** [comment]
Have also tested Russian web page with English as my device language. And in that case numbers were still read out in Russian, but some of the things were read out in English (for example / was read out as "slash"). Also, some of the text (e.g. Radio Schedule section label) was read out in broken Russian with an English accent, just as the Github issue suggested.
